.github/workflows/sweeper.yaml

```diff
@@ -9,14 +9,18 @@ permissions:
 jobs:
   sweeper:
     if: github.repository == 'aws/karpenter' || github.event_name == 'workflow_dispatch'
+    strategy:
+      fail-fast: false
+      matrix:
+        region: [us-east-2, us-west-2]
```

Comment:
Should we go ahead and add the `eu-west` region ahead of us expanding the conformance tests over there?